cdap-security-spi/src/main/java/co/cask/cdap/security/spi/authorization/Authorizer.java

```diff
@@ -54,7 +63,7 @@
    * @param action the {@link Action} being performed
    * @throws UnauthorizedException if the principal is not authorized to perform action on the entity
    */
-  void enforce(EntityId entity, Principal principal, Action action) throws UnauthorizedException;
+  void enforce(EntityId entity, Principal principal, Action action) throws Exception;
```

Comment:
I have added ``Exception`` to the javadoc. Adding ``UnauthorizedException`` to the javadoc causes a warning, because a more generic ``Exception`` is already part of the ``throws`` clause